CHANGELOG.md

```diff
@@ -10,10 +10,11 @@ All notable changes to the Pony compiler and standard library will be documented
 
 ### Added
 
+- Add FileMode.u32 ([PR #3809](https://github.com/ponylang/ponyc/pull/3809))
```

Comment:
This extra entry should probably be removed

Reply:
I dont think so, it is intentional to show that there were two PRs associated with this change.